enclave-tls/src/Makefile

```diff
@@ -24,8 +27,18 @@ HOST_LDFLAGS := -fPIE -shared -Wl,-Bsymbolic
 Lib_C_Objects := $(Lib_C_Files:.c=.o)
 NULL_TLS_WRAPPER_C_Objects := $(NULL_TLS_WRAPPER_C_FILES:.c=.o)
 NULL_ENCLAVE_QUOTE_C_Objects := $(NULL_ENCLAVE_QUOTE_C_FILES:.c=.o)
+WOLFSSL_TLS_WRAPPER_C_Objects := $(patsubst %.c, %.o, $(WOLFSSL_TLS_WRAPPER_C_FILES))
 
-all: libenclave_tls.so libtls_wrapper_null.so libenclave_quote_null.so
+WOLFSSL_CFLAGS := -DWOLFSSL_SGX_ATTESTATION -DWOLFSSL_ALWAYS_VERIFY_CB -DKEEP_PEER_CERT -DWOLFSSL_TEST_CERT -DWOLFSSL_SMALL_CERT_VERIFY -Wno-stringop-truncation
+ifdef DEBUG
+    WOLFSSL_CONFFLAGS += --enable-debug
+endif
+WOLFSSL_CONFIGURE_FLAGS := --prefix=$(shell readlink -f $(WOLFSSL_BUILD)) --enable-writedup --enable-shared  --enable-keygen --enable-certgen --enable-certext --with-pic --disable-examples --disable-crypttests --enable-aesni --enable-tlsv10 $(WOLFSSL_CONFFLAGS)
+CFLAGS += $(WOLFSSL_CFLAGS)
+
+deps := wolfssl
```

Comment:
Move this to the makefile in tls wrapper layer. wolfssl and wolfssl-sgx both depends on wolfssl, so the build of wolfssl should be triggered by wolfssl and wolfssl-sgx conditionally.